packages/legend-application-studio/src/stores/editor-state/element-editor-state/service/ServiceExecutionState.ts

```diff
@@ -754,23 +816,28 @@ export class SingleServicePureExecutionState extends ServicePureExecutionState {
   }
 
   changeExecution(): void {
-    const _execution = new PureMultiExecution(
-      this.multiExecutionKey,
-      this.execution.func,
-      this.serviceEditorState.service,
-    );
-    const _parameter = new KeyedExecutionParameter(
-      `execContext_1`,
-      this.execution.mapping,
-      this.execution.runtime,
-    );
-    _execution.executionParameters = [_parameter];
-    service_setExecution(
-      this.serviceEditorState.service,
-      _execution,
-      this.editorStore.changeDetectionState.observerContext,
-    );
-    this.serviceEditorState.resetExecutionState();
+    if (
+      this.execution.mapping !== undefined &&
```

Comment:
this.execution.mapping && this.execution.runtime

Reply:
same problem here